layouts/default.vue

```diff
@@ -60,31 +46,15 @@ export default {
   position: relative;
 }
 
-.app-footer {
-  flex: 0 0 auto;
-}
-
-#__nuxt,
-#__layout {
-  min-height: 100vh;
-}
-
-.app-container .app-header {
-  position: sticky;
-  top: 0;
-}
+// #__nuxt,
```

Comment:
I'm not sure if this is used or not, so I commented it out but left it here. 🤷‍♂️